PermissionsScrapper/PermissionsScrapper/host.json

```diff
@@ -0,0 +1,11 @@
+{
+    "version": "2.0",
+    "logging": {
+        "applicationInsights": {
```

Comment:
is this backed by an app insights instance?

Reply:
Yes.

Reply:
which one?

Reply:
From our side chat, this will need to be provisioned.